File: dynamicpdf-csharp-examples/Examples/CombinePDFs.cs
using ceTe.DynamicPDF.Merger;

namespace DynamicPDFCoreSuite.Examples
{
    class CombinePDFs
    {
        public static void Run()
        {
            CombinePDF();
            CombinePDFWithOptions();
        }

        public static void CombinePDF()
        {
            MergeDocument document = new MergeDocument(Util.GetPath("Resources/PDFs/DocumentA.pdf"));
            document.Append(Util.GetPath("Resources/PDFs/DocumentB.pdf"));
            document.Append(Util.GetPath("Resources/PDFs/DocumentC.pdf"), 1, 2);
            document.Draw(Util.GetPath("Output/CombinePDFs.pdf"));
        }

        public static void CombinePDFWithOptions()
        {
            MergeOptions options = MergeOptions.All;
            options.DocumentProperties = false;
            MergeDocument document = new MergeDocument(Util.GetPath("Resources/PDFs/DocumentA.pdf"), options);
            document.Append(Util.GetPath("Resources/PDFs/DocumentB.pdf"));
            document.Draw(Util.GetPath("Output/CombinePDFWithOptions.pdf"));
        }
    }
}

File: dynamicpdf-vbnet-examples/Examples/CombinePdfs.vb
Imports ceTe.DynamicPDF.Merger

Namespace DynamicPDFCoreSuite.Examples
    Public Class CombinePDFs

        Public Shared Sub Run()
            CombinePDF()
            CombinePDFWithOptions()
        End Sub

        Public Shared Sub CombinePDF()
            Dim document As New MergeDocument(Util.GetPath("Resources/PDFs/DocumentA.pdf"))
            document.Append(Util.GetPath("Resources/PDFs/DocumentB.pdf"))
            document.Append(Util.GetPath("Resources/PDFs/DocumentC.pdf"), 1, 2)
            document.Draw(Util.GetPath("Output/CombinePDFs.pdf"))
        End Sub

        Public Shared Sub CombinePDFWithOptions()
            Dim options As MergeOptions = MergeOptions.All
            options.DocumentProperties = False
            Dim document As New MergeDocument(Util.GetPath("Resources/PDFs/DocumentA.pdf"), options)
            document.Append(Util.GetPath("Resources/PDFs/DocumentB.pdf"))
            document.Draw(Util.GetPath("Output/CombinePDFWithOptions.pdf"))
        End Sub

    End Class
End Namespace
